tests/test_selected_columns_only.py

```diff
@@ -8,7 +8,7 @@
 
 TABLE_NAME_SELECTED_COLUMNS_ONLY = "test_selected_columns_only"
 SAMPLE_CONFIG = {
-    "sqlalchemy_url": "mysql+pymysql://root:password@localhost:3307/melty",
+    "sqlalchemy_url": f"mysql+mysqldb://root:password@127.0.0.1:3306/melty",
```

Comment:
We should explain why 127.0.0.1 instead of localhost, just saying it's because of the dialect is good enough but a comment will help someone later.

Reply:
I've added comments  to all spots where I replaced `localhost` with `127.0.0.1`.